circle.yml

```diff
@@ -10,6 +10,9 @@ dependencies:
         - sudo make dev
 test:
     override:
-        - busted -o spec/busted-print.lua --coverage spec/
+        - busted -v -o spec/busted-print.lua --coverage :
+            parallel: true
+            files:
+              - spec/**/*_spec.lua
```

Comment:
The others have a warning displayed; "Warning: Your build is configured to use 4 containers in parallel, but none of your test actions are configured to run in parallel. Consider either lowering your build parallelism, or parallelizing your tests steps."